test/rubygems/test_deprecate.rb

```diff
@@ -95,13 +95,13 @@ def execute
   end
 
   def test_deprecate_command
+    foo = FooCommand.new
+
     out, err = capture_io do
-      foo = FooCommand.new
       foo.execute
     end
 
-    assert_equal "pew pew!\n", out
-    assert_match(/NOTE: foo command is deprecated. It will be removed on or after 2099-04-01.\n/, err)
```

Comment:
Can we keep this assertion, or add a separate test to check it? I'd like the following line to be covered:

https://github.com/rubygems/rubygems/blob/3235920c22a416c95a27f5332e0b39cf72bd6532/lib/rubygems/command_manager.rb#L179

Reply:
Done @deivid-rodriguez